### ISSUE TITLE
Error with ctrl-keys

### ISSUE THREAD

**Issue by fdharamshi** [reaviz/reakeys#33]
## I'm submitting a...

<pre><code>
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When using reagraph, we use useSelection() -> which uses hotkeys, which in turn uses ctrl-keys
Now, since the change in https://github.com/reaviz/reakeys/commit/f540455c400fa9d1e62477a03d7599f3464e3ef7

I've been getting an error that says keys() is not a function at `useHotkeys.ts`:20:31

## Expected behavior

This should not throw this error. It's a blocking issue.

## What is the motivation / use case for changing the behavior?

I wanted to use reagraph library ASAP, but I am blocked because of this issue.

## Environment

<pre><code>
Libs:
- react version: 18.0.0
- reakeys version: 2.0.3

Browser:
- [x] Chrome (desktop) version latest
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 16.18.5
- Platform:  MacOS

Others:

</code></pre>

**Comment by fdharamshi** [comment]
@amcdnl please take a look^

**Comment by fdharamshi** [comment]
Hi, any updates on this? @amcdnl @evgenoid

**Comment by evgenoid** [comment]
> Hi, any updates on this? @amcdnl @evgenoid

@fdharamshi could you provide more details because I can't reproduce it?

**Comment by fdharamshi** [comment]
@evgenoid 

<img width="441" alt="image" src="https://github.com/user-attachments/assets/8f2bf52e-6bbb-4fa0-bfee-a3d7b3f30d7a">

This useSelection you see, internally uses Reakeys which uses CTRL-KEYS
And this is where I was getting the error that says keys() is not a function

**Comment by fdharamshi** [comment]
@evgenoid @amcdnl - I had to add jest.mocks in setupTests.js to bypass it for now:

```javascript
jest.mock('reagraph', () => ({
  install: jest.fn(() => {}),
}));

jest.mock('reakeys', () => ({
  keys: jest.fn(() => {}),
}));
```

But what would be the right way to fix it?

**Comment by async-jpx** [comment]
Are there any works around for this, like disabling hotkeys?

**Comment by evgenoid** [comment]
> @evgenoid
> 
> <img alt="image" width="441" src="https://private-user-images.githubusercontent.com/48595776/368568688-8f2bf52e-6bbb-4fa0-bfee-a3d7b3f30d7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0OTM0MDIsIm5iZiI6MTczNjQ5MzEwMiwicGF0aCI6Ii80ODU5NTc3Ni8zNjg1Njg2ODgtOGYyYmY1MmUtNmJiYi00ZmEwLWJmZWUtYTNkN2IzZjMwZDdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTEwVDA3MTE0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3NGVkN2M0YTEwOTMwMTc0ZjFhNDUxZDhiNDE3ZjMxZWNlYjM1NTA4NjJmN2JjYjczNDc2MjYzZWJmYzA4Y2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t6l9iuYTg6AAVQkaz5NaDOvOqVsByYIr_5feWwLfuY4">
> This useSelection you see, internally uses Reakeys which uses CTRL-KEYS And this is where I was getting the error that says keys() is not a function

Yes, and it seems the same issue for reaflow: https://github.com/reaviz/reaflow/issues/265
But I can reproduce it only in sandbox and locally with Vite all works fine for me

**Comment by amcdnl** [comment]
@evgenoid - i believe its a NextJS issue.
